src/vault_backup/backup.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import json
+import logging
+
+from lib.export_backup import (
+  backup_to_s3,
+  write_file
+)
+from lib.util import url_host_port
+from lib.vault import Vault
+
+
+def backup_secrets(
+  artsy_env,
+  vault_host,
+  vault_port,
+  kvv2_mount_point,
+  local_dir,
+  s3,
+  s3_bucket,
+  s3_prefix
+):
+  ''' export secrets '''
```

Comment:
It looks like you're not using the `black` formatter but it's configured as a pre-commit hook. Make sure you run `pre-commit install` in this repo so that you use standard formatting as there's a lot of non-standard formatting in this file.